python/python/tests/test_async_client.py

```diff
@@ -10020,6 +10020,16 @@ async def test_zscan(self, glide_client: GlideClusterClient):
         assert result[result_cursor_index] != b"0"
         assert len(result[result_collection_index]) >= 0
 
+        # Test no_scores option
+        if not await check_if_server_version_lt(glide_client, "7.9.0"):
```

Comment:
```suggestion
        if not await check_if_server_version_lt(glide_client, "8.0.0"):
```
Same here - need to find a way to make this check pass with RC